tests/routesTests/PostPortfolioTests.py

```diff
@@ -0,0 +1,157 @@
+import json
+import requests
+from unittest import main
+
+from TestConfiguration import TestConfiguration
+
+class PostPortfolioTests(TestConfiguration):
+   def setUp(self):
+      registerUrl = self.baseUrl + '/users'
+      registerBody = {
+         'firstName' : 'Dave',
+         'lastName' : 'Janzen',
+         'email' : 'dave.janzen18@gmail.com',
+         'password' : 'Stockd2g'
+      }
+      registerResponse = requests.post(url=registerUrl, data=json.dumps(registerBody), headers=self.headers)
+      self.assertEqual(registerResponse.status_code, 200)
+      self.assertEqual(self.getJson(registerResponse), None)
+
+      loginUrl = self.baseUrl + '/users/session'
+      loginBody = {
+         'email' : 'dave.janzen18@gmail.com',
+         'password' : 'Stockd2g'
+      }
+      loginResponse = requests.post(url=loginUrl, data=json.dumps(loginBody), headers=self.headers)
+      loginResponseData = self.getJson(loginResponse)
+      self.assertEqual(loginResponse.status_code, 200)
+      self.assertIsNotNone(loginResponseData['userId'])
+      self.assertIsNotNone(loginResponseData['token'])
+
+      self.userId = loginResponseData['userId']
+      self.token = loginResponseData['token']
+      self.headers['Authorization'] = 'token ' + self.token
+      self.url = self.baseUrl + '/portfolios'
+
+
+   def test_post_portfolio_soloDefaultBuyPower(self):
+      body = {
+         'name' : 'mynewportfolio',
+      }
+
+      response = requests.post(url=self.url, data=json.dumps(body), headers=self.headers)
+      responseData = self.getJson(response)
+   
+      self.assertEquals(response.status_code, 200)
+      self.assertTrue('id' in responseData)
+      self.assertTrue(responseData['id'] > 0)
+      self.assertTrue('buyPower' in responseData)
+      self.assertEquals(responseData['buyPower'], 10000)
+
+
+   def test_post_portfolio_soloCustomBuyPower(self):
```

Comment:
Do we ever do tests to see if the post made changes to the DB?

Reply:
Maybe a 501 (Not Implemented) would be more appropriate, much like the Java's UnsupportedOperationException